services/src/atdd/src/test/resources/features/mvp.1/FlowReinstall.feature

```diff
@@ -0,0 +1,64 @@
+@FREINSTALL
+Feature: Flow re-reinstalling after switch comes back up.
+
+  @MVP1.1
+  Scenario: Re-installing Ingress and Egress flows.
+
+  This scenario creates simple network and ingress/egress flows. Next this scenario turns off and on back, then checks whether
+  flows were re-installed again.
+
+    Given a clean flow topology
+    And a clean controller
+    And a clean flow topology
+    And created simple topology from two switches
+    And topology contains 2 links
+    And flow pcet creation request with 00:01:00:00:00:00:00:01 1 0 and 00:01:00:00:00:00:00:02 2 0 and 1000000 is successful
+    And flow pcet in UP state
+
+    When switch switch1 is turned off
+    Then flow pcet in DOWN state
+
+    When switch switch1 is turned on
+    Then flow pcet in UP state
+
+  @MVP1.1
+  Scenario: Re-installing transit flow when no other path is available.
+
+  This scenario created topology and builds flow through target switch. This switch will be turned off and on,
```

Comment:
Similar to above, please include a simple example of the topology, which switch is turned off, etc.